xbmc/pvr/channels/PVRChannelGroup.cpp

```diff
@@ -683,8 +703,8 @@ bool CPVRChannelGroup::IsGroupMember(int iChannelId) const
   bool bReturn(false);
   CSingleLock lock(m_critSection);
 
-  for (PVR_CHANNEL_GROUP_MEMBERS::const_iterator it = m_members.begin(); !bReturn && it != m_members.end(); ++it)
-    bReturn = (iChannelId == it->second.channel->ChannelID());
+  for (std::map<std::pair<int, int>, std::shared_ptr<PVRChannelGroupMember>>::const_iterator it = m_members.begin(); !bReturn && it != m_members.end(); ++it)
+    bReturn = (iChannelId == it->second->channel->ChannelID());
```

Comment:
get rid of `bReturn`, do early `return true;` on match => range based for loop possible